### DISCUSSION TITLE
Brazilian Portugues translation

### DISCUSSION THREAD

**Discussion by fabiomattes2016** [MarJose123/filament-lockscreen#37] [Ideas]
`<?php

return [
    'heading' => 'Bloquear Tela',
    'user_menu_title' => 'Bloquear Tela',
    'fields' => [
        'password' => 'Senha',
    ],
    'button' => [
        'switch_account' => 'Trocar de Conta',
        'submit_label' => 'Login',
    ],
    'notification' => [
        'title' => 'Falha no login',
        'message' => 'Você será redirecionado ao login devido a uma falha.',
    ],
];`

**Comment by MarJose123** [comment]
Please submit this as a PR, thanks. 🤘
